robot/Cumulus/tests/browser/crlps/total_current_year_payments.robot

```diff
@@ -0,0 +1,90 @@
+    ${filterPools}=  Create List           ${dict1}       ${dict2}
+
+    Create New Filter Setting
+    ...                                                   @{filterPools}
+    ...                                                   &{dict}
+    Enable Customizable Rollups
+    Load Page Object                                       Custom                          CustomRollupSettings
+    Navigate To Crlpsettings
+    Create New Rollup Setting
+        ...                                                Target Object=Account
+        ...                                                Target Field=This Year Payments on Past Year Pledges
+        ...                                                Description=Shows Payments made this year for pledges (Opportunities) from the prior year.
+
+        ...                                                Operation=Sum
+        ...                                                Time Frame=Years Ago
+        ...                                                Years Ago=This Year
+        ...                                                Rollup Type=Payment -> Account (Hard Credit)
+
+        ...                                                Filter Group=Old Payments
+        ...                                                Date Field=Payment: Payment Date
+
+
+    # Create an opportunity for prior year that has 6 payments associated with a contact .
+    # Ensure that Two of the payments were done the current year and one is paid last year.
+    ${opp_date} =            Get Current Date    result_format=%Y-%m-%d    increment=-365 days
+    ${payment_fields} =      Create Dictionary   Amount=833.34   NumPayments=6   Scheduledate=${opp_date}      Interval=1   Paid=3
+    ${opportunity_fields} =  Create Dictionary   Type=Donation
+    ...                                          Name=Auto Payment test $1000 Donation
+    ...                                          Amount=5000     CloseDate=${opp_date}    StageName=Closed Won    npe01__Do_Not_Automatically_Create_Payment__c=true
+    Setupdata                contact             contact_data=${contact1_fields}          opportunity_data=${opportunity_fields}    payment_data=${payment_fields}
+
+
+    Go To Page                           Details
+    ...                                  Opportunity
+    ...                                  object_id=${data}[contact_opportunity][Id]
+
+    # Navigate to the Account page and verify the new rollupfield is appearing and is showing the right amouunt
+    Go To Page                           Details                                 Account                                object_id=${data}[contact][AccountId]
+    Wait Until Loading Is Complete
+    Select Tab                                            Details
+    Navigate To And Validate Field Value                  This Year Payments on Past Year Pledges                contains         $1,667
```

Comment:
use 'Verify Rollup Field Value ' keyword instead. It has logic to recalculate rollups and refresh in cases where the update takes time. I have noticied falkyness by just using navigate keyword